AirCasting/SessionsSynchronization/Controller/SessionSynchronizer.swift

```diff
@@ -25,6 +25,9 @@ protocol SessionSynchronizer {
     func stopSynchronization()
     /// A plugin point for anyone interested in generated errors
     var errorStream: SessionSynchronizerErrorStream? { get set }
+    
+    /// Triggers downloading single session
+    func downloadSingleSession(sessionUUID: SessionUUID, completion: @escaping () -> Void)
```

Comment:
hmm, I'm not sure that this method belongs to the synchronizer. It's probably ok that it is implemented inside `SessionSynchronizationController`, but I think it needs a separate `protocol`. There is this rule called "interface segregation principle" (ISP, the I of the SOLID) which says that no code should be forced to depend on methods it doesn't call. And adding this method violates this principle because inside the unit that uses this particular method no other methods of this protocol are called and in the code that uses the other methods this one won't get called. So this tells us that it actually is something else and deserves a separate interface (protocol).

I'd recommend creating it for the sake of keeping this part of the codebase clean. Also consider writing a unit test for this one (there are tests for the `SessionSynchronizationController` so maybe you can try to look at how those work and give it a shot?)